### DISCUSSION TITLE
How do I force the Quick Selector to connect to an address using a different profile type other than SSH?

### DISCUSSION THREAD

**Discussion by BallisticCrux** [Eugeny/tabby#8679] [Q&A] [answered]
Thought I would be able to type "telnet 1.2.3.4" into the quick selector or any other keyword but every time it defaults to connecting via SSH. I've looked through the documentation and cannot find if this is a feature or if its not supported and I have to manually create a specific profile for every connection type other than SSH. Attempted asking for clarification on an open query but it was not elaborated on.

![image](https://github.com/Eugeny/tabby/assets/3671552/5d20e834-a002-4d84-ab5e-bb4fcc26c58d)

**Comment by Eugeny** [comment, marked as answer]
This is not possible unless you disable the SSH plugin unfortunately - SSH is hardcoded to have higher priority as the "connect to..." provider.

There are separate type-specific profile selector windows that you can trigger with a hotkey (e.g. the "Show Telnet profile selector" hotkey), but these lack the "connect to" option - I've opened #8680 to add it
